examples/mixertest/mixertest.c

```diff
@@ -70,7 +70,7 @@ int main(void) {
 			music_frequency /= 1.1;
 			mixer_ch_set_freq(CHANNEL_MUSIC, music_frequency);
 		}
-		if (music && music_frequency <= 128000 && ckeys.c[0].R) {
+		if (music && music_frequency*1.1 <= 128000 && ckeys.c[0].R) {
```

Comment:
What was this for?

Reply:
See the commit message: I found this small bug in mixertest. The test is configuring the mixer for max 128Khz playback on that channel, but then the code was wrong and was playing a frequency higher than that. This started failing with assert with this PR that contains a commit where the frequency limit is enforced via assert, rather than crossing fingers and hoping the samplebuffer is big enough anyway.

Reply:
Might be nice to squash those two commits if that's ok